frontends/p4/validateParsedProgram.h

```diff
@@ -81,7 +83,7 @@ class ValidateParsedProgram final : public Inspector {
                            control->getConstructorParameters());
     }
     void postorder(const IR::P4Parser *parser) override {
-        auto start = parser->states.getDeclaration("start");
+        auto start = parser->states.getDeclaration("start"_cs);
```

Comment:
Maybe get it from the `P4CoreLibrary`?

Reply:
I do not see anything related there. Could you please elaborate?

Reply:
Oh, I thouht the start state name is there but it is in ParserState actually

```suggestion
        auto start = parser->states.getDeclaration(IR::ParserState::start);